Dockerfile

```diff
@@ -1,7 +1,15 @@
-FROM golang:1.9
-MAINTAINER Leif Madsen <leif@leifmadsen.com>
+ARG GO_VERSION=1.13
 
-RUN go get github.com/cloudwan/gohan
+
+FROM golang:$GO_VERSION-alpine
+RUN apk add --no-cache g++
```

Comment:
I assume that this is needed for cgo sqlite, right? If only for that then probably gcc+libc-dev would suffice.

Reply:
Yes, you are right, done. Good suggestion, thanks!